src/api/index.ts

```diff
@@ -22,6 +22,22 @@ class API {
     return this.request<any>('POST', '/wallets', args);
   }
 
+  addSigner(args: T.AddSignerArguments) {
+    return this.request<any>('POST', '/signers', args);
+  }
+
+  promptPin(args: T.PromptPinArguments) {
+    return this.request<any>('POST', '/prompt', args)
+  }
+
+  enterPin(args: T.EnterPinArguments) {
+    return this.request<any>('POST', '/unlock', args)
+  }
+
+  deletePrompt() {
+    return this.request<any>('DELETE', '/prompt',)
+  }
```

Comment:
We should type all of these responses at some point, `<any>` was only meant as a placeholder.